exercises/practice/rotational-cipher/.meta/proof.ci.wren

```diff
@@ -0,0 +1,13 @@
+class RotationalCipher {
+  static rotate(text, shiftKey) {
+    var rotateCodePoint = Fn.new {|codePoint|
+      if ((65..90).contains(codePoint)) {
+        codePoint = (codePoint - 65 + shiftKey) % 26 + 65
+      } else if ((97..122).contains(codePoint)) {
+        codePoint = (codePoint - 97 + shiftKey) % 26 + 97
+      }
+      return String.fromCodePoint(codePoint)
+    }
+    return text.codePoints.map(rotateCodePoint).join()
+  }
```

Comment:
```suggestion
      String.fromCodePoint(codePoint)
    }
    text.codePoints.map(rotateCodePoint).join()
  }
```

Like in Ruby return is not necessary in most cases.

Reply:
I have now made `rotate` a single-line function, with no `return` keyword.

If we remove the `return` before `String.fromCodePoint(codePoint)`  I observe results like `nullnullnullnullnull`

Reply:
This is discussed in https://wren.io/syntax.html#blocks -- as soon as there's a newline after the opening `{`, you need to explicitly `return`